packages/react/.gitignore

```diff
@@ -21,22 +21,25 @@ bower_components/
 node_modules/
 package-lock.json
 npm-debug.log
+npm-debug.log*
 yarn.lock
+yarn-debug.log*
 yarn-error.log
+yarn-error.log*
+/.pnp
+.pnp.js
 
 # Atom Sync
 .remote-sync.json
 
-# Package Minifications
-packages/*/src/**/*.min.js
-packages/*/src/**/*.min.css
-packages/*/src/**/*.min.html
-packages/*/src/**/*.css.map
-
-# Package Compilations
-packages/angular/**/*.js
-packages/angular/**/*.css
-packages/boot/**/*.js
-packages/core/**/*.js
-packages/react/**/*.js
-packages/react/**/*.css
```

Comment:
Won't this start to add in all the auto generated js and css files? (can possible cause future merge conflicts)